newIDE/app/src/EventsSheet/index.js

```diff
@@ -107,6 +108,12 @@ type State = {|
   showSearchPanel: boolean,
   searchResults: ?Array<gdBaseEvent>,
   searchFocusOffset: ?number,
+
+  allEventsMetadata: Array<{
```

Comment:
Replace `import { enumerateEventsMetadata } from './EnumerateEventsMetadata'; `  by `import { enumerateEventsMetadata, type EventMetadata } from './EnumerateEventsMetadata';`

then you are able to write:

`allEventsMetadata: Array<EventMetadata>`

and anyone using `EnumerateEventsMetadata.js` in the future will be able to both use the function and get the typing without to write the typing manually again :)

Reply:
ah nice, I didn't realize you can import the type like that too. Very convenient :D 
I applied all changes, but will need to do the commit later today when I am home

Reply:
Cool! :) No worries take your time.